android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/PwoDiscoveryService.java

```diff
@@ -117,12 +139,11 @@ private void initialize() {
     for (PwoDiscoverer pwoDiscoverer : mPwoDiscoverers) {
       pwoDiscoverer.setCallback(this);
     }
+    mPwoResponseCallbacks = new ArrayList<>();
```

Comment:
Minor point here, but some of these members are initialized above. Would it make sense to consolidate the variable inits there (since many other variables are initialized there also)?